### PR TITLE
Removes loot from Syndicate and Spaceport simple mobs.

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl:
del: Removed loot drops from Syndicate Simple mobs.
/:cl:

[why]: # _Xenobiologists_ Orion Trial emaggers spawning them and abusing sentience/non-aggression potion to get Energy shields and C20rs every round. Literally away mission era 2.0.

### CODE REVIEW

**Comment by vcordie** [comment]
Not how that worked. You had to go throughn an orion trail machine that literally had a chance to INSTANTLY KILL YOU for a chance to get c20-rs and eshields. You need an emag, and its not every round. Fucking rip dude.

**Comment by vcordie** [comment]
in any case it'd be more cost effective for you to just buy a .357 revolver

**Comment by kevinz000** [comment]
Uh no.
Make xenobio drop no loot varients. These are used in other places don't fucking blanket nerf because people are being dumb.

**Comment by LanCartwright** [comment]
Yeah no 6tc for Energy shield and C20rs no more.

**Comment by kevinz000** [comment]
@vcordie THIS ISNT THE TRAIL MOBS YOU DUMBFUCK

**Comment by kevinz000** [comment]
Or you can fuck off jmad :^)

**Comment by vcordie** [comment]
I KNOW. HE'S NERFING SOME RETARDED OTHER MOBS

**Comment by vcordie** [comment]
like, what the even fuck is this

**Comment by LanCartwright** [comment]
those can go too

**Comment by vcordie** [comment]
aww muffin, did you die in one of the extremely rare rounds where someone has both the knowledge and the means to do this?

**Comment by LanCartwright** [comment]
No actually, I killed you.

**Comment by kevinz000** [comment]
Vcordie don't abuse shit to the point people get annoyed.

**Comment by kevinz000** [comment]
Ma44 needs to take a cup of STFU too.

**Comment by kevinz000** [comment]
And you know not brag about abusing shit to the point people want to nerf it.

**Comment by Repukan** [comment]
Notice how RnD mains are the ones opposing this

**Comment by Bawhoppen** [comment]
:+1:  We don't need game-y dumb things like this even if they don't typically cause a balance concern.

**Comment by FantasticFwoosh** [comment]
Going to namedrop @MMMiracles here as the creator of the exploited battlecruiser away mission & the current pirates vs syndicate caravan space ruin which has the same issues with space explorers coming back from there with this loot.

**Comment by ma44** [comment]
@kevinz000 it's called "talking about a feature I never use"

**Comment by MMMiracles** [comment]
I never created that away mission what are you even talking about.

**Comment by Iamgoofball** [comment]
R&Dtards get BTFO

**Comment by ma44** [comment]
What's the point of this being related to xenobiology

No xenomobs really give any decent loot

**Comment by Iamgoofball** [comment]
R&Dtards btfo

**Comment by loqoman** [comment]
So what does the PR mean? Does emagging the Orion Trail give OP loot?

I´m quite lost here. Someone dumb it down for me please!

👍

**Comment by MMMiracles** [comment]
@loqoman From the sounds of it, a traitor scientist in xenobio realized he could emag the Orion Trial arcade machine and use the fruits of his labor to make them non-aggressive and/or sentient and loyal to him, essentially granting him free C-20 SMGs and eshields.

In better terms: man gets salty because traitor does traitor thing with current game mechanics and now he wants to nerf it

**Comment by Tacolizard** [comment]
@loqoman emagging an orion trail machine spawns syndie-equipped orion trail 'guards'

**Comment by Balcore** [comment]
As a hyper lurker I can firmly say 👎

Reasoning being that this PR will impact all Syndie NPC's in its current form. As one of the stranger beings that actually likes exploring the debris field on slow shifts, the Syndicate salvage team (if it spawns) is a juicy target for some cool loot, but it also isn't free and poses some challenge to get.

This PR eliminates the drive to actually fight the Syndies there and guts one of the more interesting ruins in the Debris Field.
 
>inb4 shut up n3wguy b1gb0ys are talkun

**Comment by Iamgoofball** [comment]
We can always put some of this shit out on the field itself to avoid losing it entirely.

**Comment by Balcore** [comment]
@Iamgoofball Do you mean caching Syndie loot in the debris field and other ruins?

Ex: Salvage team members no longer drops loot but now on the salvage ships nearby there's a chance for syndicate loot to spawn in box's/lockers?

**Comment by Iamgoofball** [comment]
Yeah, something like that. Maybe set the syndies to guard that area in specific too.

**Comment by loqoman** [comment]
@MMMiracles 
So emagging the machine makes it sentient ? 

Also, since when do the arcade machines contain lethal weapons?

**Comment by Iamgoofball** [comment]
@loqoman 

So you know how when you emag arcade machines, they turn dangerous, right?

Emagging the boss fight arcade makes you fight Cuban Pete, if you win you get a free maxcap bomb and a free hat.

Emagging Orion Trail will make the things you do in game actually happen, so you can use this to spawn syndicate simple mobs to attack you. Killing them would drop syndie loot, which you could then R&D decon for points/use for yourself for FREE TRAITOR GEAR

**Comment by Balcore** [comment]
@Goof Didn't the Cuban Pete minibomb get mega nerfed? 

Last I checked it was nowhere close to maxcap

**Comment by Iamgoofball** [comment]
that might be as a result of it not being updated to fit new changes

**Comment by WJohn** [comment]
There's a reason these were removed from xenobio's gold cores.

**Comment by PKPenguin321** [comment]
Leave Orion trail alone

**Comment by ma44** [comment]
>not just making them not be able to be made friendly

**Comment by loqoman** [comment]
@Iamgoofball 
Thanks a lot. I can now follow the conversation.

**Comment by Iamgoofball** [comment]
No problem friend

**Comment by RemieRichards** [comment]
I won't be accepting this fix.
They are intended to drop loot.
What they are NOT intended to do is be sentience potioned, and fixing that is a one variable change (though I feel it shouldn't work on any "human" like simple animals, so that'd be a few more lines for each of those)

**Comment by juanototo2** [comment]
>congrats you earned it

>stand behind window with laser or abuse simple mob AI

l e l

**Comment by Cheridan** [comment]
>if you cheesed them: go and stay go.

Admins don't and have never banned for cheese, the only way to prevent exploit scenarios like this is to fix them in the code.

**Comment by RemieRichards** [comment]
@Cheridan Which is literally what I proposed? I'm saying that if you are resorting to cheese you're a shitty player, not that you deserve to be banned.

**Comment by kevinz000** [comment]
more downvotes than upboats lmao

**Comment by FantasticFwoosh** [comment]
You can just workabout a sentience potion by racechanging yourself to a cult golem and dominating them.

Either that or just sentient a statue, emag it in safe conditions and let the virtually unkillable mob deal with it.

**Comment by LanCartwright** [comment]
There simply isn't any room for changes here. I could possible remove the eshield and C20r, but that's literally the entire problem of the loot. I can do that if you'd like, but there is the problem of still leaving behind an Elite Syndicate hardsuit.

**Comment by kevinz000** [comment]
yes there is
it's called close PR.

**Review by Cyberboss — APPROVED** [review]

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -1050,25 +1050,8 @@
...
-	suit = /obj/item/clothing/suit/armor/vest
-	shoes = /obj/item/clothing/shoes/combat
-	gloves = /obj/item/clothing/gloves/combat
-	ears = /obj/item/device/radio/headset
-	mask = /obj/item/clothing/mask/gas
-	head = /obj/item/clothing/head/helmet/swat
-	back = /obj/item/weapon/storage/backpack
-	id = /obj/item/weapon/card/id
+	loot = list()
+	del_on_death = true
```
Comment (Cyberboss) [inline code/game/machinery/computer/arcade.dm]: TURE

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -1050,25 +1050,8 @@
...
-	uniform = /obj/item/clothing/under/syndicate
-	suit = /obj/item/clothing/suit/armor/vest
-	shoes = /obj/item/clothing/shoes/combat
-	gloves = /obj/item/clothing/gloves/combat
-	ears = /obj/item/device/radio/headset
-	mask = /obj/item/clothing/mask/gas
-	head = /obj/item/clothing/head/helmet/swat
-	back = /obj/item/weapon/storage/backpack
-	id = /obj/item/weapon/card/id
+	loot = list()
```
Comment (Cyberboss) [inline code/game/machinery/computer/arcade.dm]: Why'd you remove the gibs?

**Review by Cyberboss — APPROVED** [review]

**Comment by KorPhaeron** [comment]
I've been seeing assistants in elite hardsuits every other round and science is setting up safe mob farms in the testing range (letting space kill them)

Another fun feature abused forever until it died

**Comment by KorPhaeron** [comment]
If anyone can think of a sane middle ground it can come back

**Comment by Lumipharon** [comment]
Make it that you have to win the game or get gibbed, and if you manage to win you come out with a random syndie item or some shit?

**Comment by KorPhaeron** [comment]
@Lumipharon People screwdriver the machine to reset it when losing

**Comment by Lumipharon** [comment]
Can you just them blow the fuck up or some shit if they get forced out of the game in anyway? Powergamers mang.
(would be hilarious if a powersink caused someone to gib by cutting power though)

**Pull request merged**